src/main/java/sic/service/impl/EmpresaServiceImpl.java

```diff
@@ -130,16 +130,20 @@ public Empresa guardar(Empresa empresa) {
 
     @Override
     @Transactional
-    public void actualizar(Empresa empresa) {
-        validarOperacion(TipoDeOperacion.ACTUALIZACION, empresa);
-        empresaRepository.save(empresa);
+    public void actualizar(Empresa empresaParaActualizar, Empresa empresaPersistida) {
```

Comment:
@k31kur0n0 
Agregar { } al IF para mejor lectura (es demasiada larga la expresion)
Aplicar format
Agregar this. delante de validarOperacion para indicar que es un metodo de la misma class